docs/source/apps/registration.md

```diff
@@ -62,15 +62,15 @@ The endpoints shown there are directly generated based on the uploaded openAPI f
 More information on swagger is available in the official [swagger documentation](https://swagger.io/docs/).
 
 ```{Important}
-When communicating with the registered application via the MarketPlace, the query and path parameters of your application endpoint should always be sent as query parameters.
+When communicating with the registered application via the MarketPlace, the _query_ and _path_ parameters of your application endpoint should all be sent as _query_ parameters.
 
 __Example 1__<br>
 Application endpoint:<br><font style="color:gray">https://my-app.com/simulation/run/4f384c28-8671-4c7d-a6cf-22d127fc0866</font><br>
 MarketPlace endpoint:<br><font style="color:gray">https://www.materials-marketplace.eu/mp-api/proxy/<CLIENT_ID>/start_transformation?transformationId=4f384c28-8671-4c7d-a6cf-22d127fc0866</font>
```

Comment:
The path to the proxy should also be updated, right?

Reply:
```suggestion
MarketPlace endpoint:<br><font style="color:gray">https://www.materials-marketplace.eu/api/applications/proxy/<CLIENT_ID>/updateTransformation?transformationId=4f384c28-8671-4c7d-a6cf-22d127fc0866</font>
```

Reply:
However, since the new way to start a transformation is to patch the status to `RUNNING`, maybe we can use another capability for the example (like `getTransformation`)